components/Footer.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import Link from "next/link";
+import Image from "next/image";
+
+const FooterContainer = styled.footer`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.4rem;
+  background-color: #e5f7eb;
+  position: sticky;
+  bottom: 0;
+  left: 0;
+  width: 98%;
+  z-index: 1;
+  border-top: 1px solid #ccc;
+  gap: 2rem;
+`;
+
+const TabBarItem = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  cursor: pointer;
+  transition: opacity 0.3s, filter 0.3s;
+
+  &:hover {
+    opacity: 0.8;
+  }
+
+  &.selected {
+    color: darkgreen;
+  }
+`;
+
+const Footer = () => {
+  const [selectedTab, setSelectedTab] = useState("");
+
+  const handleTabClick = (tabName) => {
+    setSelectedTab(tabName);
+  };
+
+  return (
+    <FooterContainer>
+      <TabBarItem
+        className={selectedTab === "home" ? "selected" : ""}
+        onClick={() => handleTabClick("home")}
```

Comment:
Do you really need this? Because when a page is selected and the footer is visible, nothing changes?

Reply:
No, I had previously inverted the colors when the tab was pressed but it looked strange, will remove, thank you!